<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationOptionsDialog</class>
 <widget class="QDialog" name="StationOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="timeGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Station Time Interval</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="endtimeDateTimeEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="endtimeLabel">
        <property name="text">
         <string>End Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="starttimeDateTimeEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="starttimeLabel">
        <property name="text">
         <string>Start Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="snclGroupBox">
     <property name="title">
      <string>SNCL</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QLineEdit" name="stationLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="networkLabel">
        <property name="text">
         <string>Network</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="networkLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="stationLabel">
        <property name="text">
         <string>Station</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="locationLineEdit"/>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="channelLineEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Locaion</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="channelLabel">
        <property name="text">
         <string>Channel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="locationGroupBox">
     <property name="title">
      <string>Location Box</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLineEdit" name="locationRangeWestLineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="locationRangeEastLineEdit"/>
      </item>
      <item row="2" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="centerLabel">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="locationRangeSouthLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="locationRangeNorthLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close Window</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
